pkg/repository/testworkflow/mongo.go

```diff
@@ -60,9 +63,39 @@ type MongoRepositoryOpt func(*MongoRepository)
 
 func (r *MongoRepository) Get(ctx context.Context, id string) (result testkube.TestWorkflowExecution, err error) {
 	err = r.Coll.FindOne(ctx, bson.M{"$or": bson.A{bson.M{"id": id}, bson.M{"name": id}}}).Decode(&result)
+
+	if result.ResolvedWorkflow != nil && result.ResolvedWorkflow.Spec != nil && result.ConfigParams != nil {
+		r.populateConfigParams(result.ResolvedWorkflow, result.ConfigParams)
+	}
+
 	return *result.UnscapeDots(), err
 }
 
+func (r *MongoRepository) populateConfigParams(resolvedWorkflow *testkube.TestWorkflow, configParams map[string]testkube.TestWorkflowExecutionConfigValue) {
```

Comment:
Not important: I don't think it should be part of this struct, as it's completely independent